### ISSUE TITLE
Warning 'onRequestClose'

### ISSUE THREAD

**Issue by valeriaabo** [tibbus/react-native-fit#1]
Im using FitModal and Im getting the error message:

> **Warning: Failed prop type: The prop 'onRequestClose' is marked as required in 'Modal' but value is 'undefined'.**

Does anyone know how to fix it? Please! :(

**Comment by tibbus** [comment]
Hi @valeriaabo I think this is because you are on Android ? I tested it only for iOS.

I will look in to it though.

**Comment by tibbus** [comment]
I have added a new release [1.1.1](https://github.com/tibbus/react-native-fit/releases/tag/1.1.1) which fixes this issue.
If you upgrade to that version it should fix the Warning.

**Comment by valeriaabo** [comment]
Thanks a lot, it works but now Im getting another error with the react-native-fetch-blob, and the top bar is not hidding when the modal opens :(

**Comment by tibbus** [comment]
the `react-native-fetch-blob` is a package that was removed, I think it should be fixed if you remove the react-native-fit package and install it again, also run `react-native link` to updates the links

**Comment by valeriaabo** [comment]
@tibbus 
Thanks the errors are gone, just one thing, the top status bar ( where you see the battery, hour, etc ) is supposed to be shown when the modal is open? Im testing on android

**Comment by tibbus** [comment]
No, there is some code to hide it, but tested only on iOS, should also work on Android but will look into it to see what can be wrong.

**Comment by valeriaabo** [comment]
OK thanks a lot! I'll be patiently waiting :D
